net-dns/maradns/maradns-3.5.0036.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..11} )
+inherit python-any-r1 systemd toolchain-funcs
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="https://maradns.samiam.org"
+SRC_URI="https://maradns.samiam.org/download/${PV%.*}/${PV}/${P}.tar.xz"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="examples"
+
+BDEPEND="${PYTHON_DEPS}
+	dev-lang/perl"
+DEPEND="
+	acct-group/maradns
+	acct-user/duende
+	acct-user/maradns"
+RDEPEND="${DEPEND}"
+
+#PATCHES=(
+	#"${FILESDIR}"/${PN}-3.5.0022-flags.patch
```

Comment:
It was likely just me being reckless and pushing without verifying, rebased the patch to work in the latest commit!

Reply:
This is *incredibly* broken.

The stable ebuild doesn't compile anymore because the patch doesn't apply, as it shouldn't since the patch was modified in place to apply against a different version instead.

0c153250617ed0d50f1ee4ac1e7aa53bad9e87af